Codes/structs.exs

```diff
@@ -0,0 +1,6 @@
+defmodule Perfil do
+  defstruct [:name, :age, :country, :food] #This is something like a class, I can create maps based
+                                            #of this module. With this elements as a key, no more or less
+end
+map_Perfil= %Perfil{age: 22,country: "Colombia",food: "Huevo",name: "Andres"} # Create my map struct
```

Comment:
Same here you should use snake_case convention, I suggest you to take a look at this https://hexdocs.pm/elixir/main/naming-conventions.html